src/libraries/Common/src/Interop/OSX/Interop.libobjc.cs

```diff
@@ -46,14 +46,17 @@ internal static Version GetOperatingSystemVersion()
                 }
             }
 
-            if (major == 10 && minor == 16)
```

Comment:
We don't need the Big Sur 10.16 => 11 mapping any more since macOS 11 is far out of support.